src/lndUtils.ts

```diff
@@ -184,6 +185,24 @@ export const getRoutingFees = async ({
   return _.map(feePerDate, (v, k) => ({ [k]: v }))
 }
 
+export const getInvoiceAttempt = async ({ lnd, id }) => {
```

Comment:
maybe is better just getInvoice and only returnning the invoice. If return an invoice is true, if is null is false and any other case throw the error (just like now).

```js
try {
  return await getInvoice({ lnd, id })
} catch (err) {
  const invoiceNotFound = "unable to locate invoice"
  if (err.length === 3 && err[2]?.err?.details === invoiceNotFound) {
    return null
  }
  // must be wrapped error?
  throw err
}
```

Reply:
this is indeed cleaner